docker-compose-sgx.yaml

```diff
@@ -15,33 +15,49 @@
 version: '3.5'
 
 services:
-  tcf:
-    container_name: tcf
-    image: tcf-dev
+  avalon-shell:
+    container_name: avalon-shell
+    image: avalon-shell-dev
     build:
       context: .
-      dockerfile: ./docker/Dockerfile.tcf-dev
+      dockerfile: ./docker/Dockerfile
       args:
         - http_proxy
         - https_proxy
         - no_proxy
-        - SGX_MODE=HW
-        - TCF_DEBUG_BUILD=${TCF_DEBUG_BUILD:-}
         - DISPLAY=${DISPLAY:-}
-        - MAKECLEAN=${MAKECLEAN:-1}
     volumes:
       # Below volume mappings are required for DISPLAY settings by heart disease GUI client
       - /tmp/.X11-unix:/tmp/.X11-unix
       - ~/.Xauthority:/root/.Xauthority
+    working_dir: "/project/avalon"
+    entrypoint: bash -c "tail -f /dev/null"
+    stop_signal: SIGKILL
+    depends_on:
+      - avalon-lmdb
```

Comment:
Is avalon-shell is used to run clients ? Then it should depend on other avalon components like listener and enclave manager

Reply:
Yes, its used to run clients and tests. Client depends on the db to be up and running along with enclave manager and listener. But adding enclave manager and listener as dependencies make sense because they are dependent on lmdb service to be up. I will make the change.